src/lib/ekoEmbed/v2.js

```diff
@@ -25,6 +25,12 @@ class EkoEmbedV2 {
         embedParams.embedid = this.iframe.id;
         embedParams.events = options.events.join(',');
 
+        if (typeof options.csp === 'object') {
```

Comment:
Change this line to `if (options.csp && typeof options.csp === 'object') {`
This because `null` would also evaluate to true (`typeof null === 'object'`).